Dockerfile-linux

```diff
@@ -0,0 +1,53 @@
+FROM ubuntu:xenial
+MAINTAINER Pranjal <pranjal@brave.com>
+
+RUN \
+  apt-get update && \
+  apt-get install -y build-essential curl autotools-dev automake gnupg2 && \
+  curl -fsSL "https://zlib.net/zlib-1.2.11.tar.gz" -o zlib-1.2.11.tar.gz && \
+  echo "c3e5e9fdd5004dcb542feda5ee4f0ff0744628baf8ed2dd5d66f8ca1197cb1a1  zlib-1.2.11.tar.gz" | shasum -a 256 -c - && \ 
+  tar -zxvf zlib-1.2.11.tar.gz && \
+  cd zlib-1.2.11 && \
+  ./configure --prefix=$PWD/install && \
+  make && make install && \
+  cd ..
```

Comment:
No need for `cd ..` here, I think -- can't imagine RUN has persistent effects on working directory.